app/lib/constants/site_config.rb

```diff
@@ -170,15 +170,15 @@ module SiteConfig
         placeholder: SVG_PLACEHOLDER
       },
       logo_png: {
-        description: "Minimum 1024px, used for PWA etc.",
+        description: "Used as a fallback to the SVG. Recommended minimum of 192x192px for PWA support",
```

Comment:
the 1024px has to be a typo right? I'm not sure if there really is a required size at all but chose 192x192px because that's the ideal size to support [PWA](https://web.dev/add-manifest/). Does that sound alright @benhalpern ?

Reply:
I think we need a higher minimum.

Right now 1024px is chosen because we use it with sizes up to 512 and it's always much better to have a bigger logo we can downsize vs the other way around. 512 is probably good enough but I wouldn't go lower than that.

Reply:
Sounds good. I'm changing it to 512 x 512px
```suggestion
        description: "Used as a fallback to the SVG. Recommended minimum of 512x512px for PWA support",
```